magefile.go

```diff
@@ -109,8 +109,8 @@ func AddLicenseHeaders() error {
 		licenser(
 			licenser.License("ASL2"),
 			licenser.Exclude("x-pack"),
-			licenser.Exclude("generator/beat/{beat}"),
-			licenser.Exclude("generator/metricbeat/{beat}"),
+			licenser.Exclude("generator/_templates/beat/{beat}"),
+			licenser.Exclude("generator/_templates/metricbeat/{beat}"),
```

Comment:
Were they moved in a previous template?

Reply:
Yes. It has to be moved, otherwise `go test` and co. are trying to run the code. But if the folder name starts with `_`, `go` skips it.
I only checked if `go test ./...` is passing. That's how I missed adjusting the paths.